### PR TITLE
Added Jetbrains to Company list

### PR DESCRIPTION
**Pull request by uddeshyatyagi**

### CODE REVIEW

**Comment by uddeshyatyagi** [comment]
Hi @mattjegan,please review my PR

**Comment by koppor** [comment]
We aim for making this list more awesome. Can you provide why the participation of JetBrains is awesome?

Please also make your contribution even more awesome by spelling "JetBrains" correctly.

**Comment by uddeshyatyagi** [comment]
JetBrains supports non-commercial **_open source_** projects by providing core project contributors with a set of best-in-class developer tools free of charge.
It is built on the IntelliJ Platform and includes its sources. JetBrains released both under Apache License 2.0.In 2010, Android support became a part of the Community Edition,and two years later Google announced its Android Studio, the IDE for mobile development on Android platform built on the Community Edition of IntelliJ IDEA and an official alternative to Eclipse Android Developer Tool.In June 2015, it was announced that the support of Eclipse ADT would be discontinued making Android Studio the official tool for Android App development.

**Comment by koppor** [comment]
Thank you for the text proposal. The first sentence is a copy'n'paste from https://www.jetbrains.com/community/opensource/.

I would like to one sentence after the JetBrains entry.

Maybe:

> JetBrains offers a wide range of developer IDEs. For Hacktoberfest, it makes them available for free for open source contributors independently of the contributor being a permanent contributor to an open source project (which is covered by their [Free Licenses Program](https://www.jetbrains.com/community/opensource/#support))

**Comment by uddeshyatyagi** [comment]
@mattjegan,I did copy 'n' past so that user come to know exactly what JetBrains says about it to the world and i didn't want anyone to mis-interpret my words in any manner.
Thank you :)

File: readme.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ Here you find projects that were made especially for hacktoberfest.
 - [Grafana Labs](https://github.com/grafana/grafana)
 - [Hasura](https://blog.hasura.io/hasura-joins-hacktoberfest-2019/)
 - [JetBrains](https://www.jetbrains.com/lp/hacktoberfest-2020/)
+```JetBrains offers a wide range of developer IDEs. For Hacktoberfest, it makes them available for free for open source contributors independently of the contributor being a permanent contributor to an open source project (which is covered by their Free Licenses Program)```
```
Comment (koppor) [inline readme.md:133]: This is not the format for an awesome-list. Are you aware of some other awesome lists such as https://github.com/sindresorhus/awesome#contents?

**Comment by uddeshyatyagi** [comment]
So where was i supposed to add about JetBrains?I didn't found any suitable file or directory to do so!

File: readme.md [hunk 2]
```diff
@@ -129,6 +129,7 @@ Here you find projects that were made especially for hacktoberfest.
 - [Gatsby](https://github.com/gatsbyjs/store.gatsbyjs.org)
 - [Grafana Labs](https://github.com/grafana/grafana)
 - [Hasura](https://blog.hasura.io/hasura-joins-hacktoberfest-2019/)
+- [JetBrains](https://www.jetbrains.com/lp/hacktoberfest-2020/)-JetBrains offers a wide range of developer IDEs. For Hacktoberfest, it makes them available for free for open source contributors independently of the contributor being a permanent contributor to an open source project (which is covered by their Free Licenses Program)
```
Comment (koppor) [inline readme.md:132]: I checked OUR contribution guide. It states:

> Add a description to your proposal. See sindresorhus/awesome@master/pull_request_template.md for details.

This file contains examples. I think, we are following more or less these examples.

However, did you see that there is a space before and after the dash? Can you please add the two spaces?

I also forgot the final dot at the sentence.

```suggestion
- [JetBrains](https://www.jetbrains.com/lp/hacktoberfest-2020/) - JetBrains offers a wide range of developer IDEs. For Hacktoberfest, it makes them available for free for open source contributors independently of the contributor being a permanent contributor to an open source project (which is covered by their Free Licenses Program).
```

**Comment by koppor** [comment]
I fixed it by myself and committed. Thank you for the proposal. This is a good start for hacktoberfest. Please contribute to other larger OSS projects (being used by many people).

**Pull request merged**